tox.ini

```diff
@@ -8,5 +8,5 @@ deps =
   py37-mypy: mypy
 
 commands =
-  py37-lint: pylint opentelemetry-api/opentelemetry/
-  py37-mypy: mypy opentelemetry-api/opentelemetry/
+  py37-lint: pylint opentelemetry-api/src/opentelemetry/
```

Comment:
@c24t do you think it's good idea to run lint on `opentelemetry-api` (which includes `setup.py`, tests, examples)?

Reply:
Sure, if it ends up being easier to blacklist dirs that shouldn't be linted than to whitelist every one that should.